Mathlib/Algebra/Exact.lean

```diff
@@ -51,6 +51,14 @@ lemma Exact.of_comp_of_mem_range [Zero P] (h1 : g ∘ f = 0)
   fun y => Iff.intro (h2 y) <|
     Exists.rec ((forall_apply_eq_imp_iff (p := (g · = 0))).mpr (congrFun h1) y)
 
+lemma Exact.comp_injective [Zero P] [Zero P'] (exact : Exact f g)
```

Comment:
We have `Injective.comp_exact_iff_exact`. Do you need the non-linearMap version?

Reply:
Not really, but somebody might do. I am not sure if I should delete it

Reply:
I think we can keep it.